packages/fuel-indexer-tests/tests/integration/web_api_postgres.rs

```diff
@@ -129,20 +133,25 @@ async fn test_asset_upload_endpoint_properly_adds_assets_to_database_postgres()
 
     let client = http_client();
     let resp = client
-        .post("http://127.0.0.1:29987/api/index/fuel_indexer_test/index1")
+        .post("http://127.0.0.1:29987/api/index/test_namespace/simple_wasm_executor")
```

Comment:
Why this change?

Reply:
If you run this test in sequence with the other tests (with no database refresh), then the test will fail as the assets are already present in the database. Since we're already using the `simple_wasm` assets as test uploads, I changed the route to match the manifest details in order to prevent any collisions.